src/components/Homepage.jsx

```diff
@@ -0,0 +1,47 @@
+import {React, useEffect, useState } from 'react'
+import { Link } from "react-router-dom"
+import { getAllArticles } from '../utils/api-requests'
+
+export default function Homepage (){
+
+    const [listOfArticles, setListOfArticles] = useState([])
+    
+    useEffect(() => {
+        getAllArticles()
+            .then(({ articles }) => {
+                console.log(articles)
+                setListOfArticles(articles)
+            })
+    }, [])
+    
+    return (
+        <>
+        <ul className="articles-list">
+            {listOfArticles.map((article) => {
+                return <div className="article-card" key={article.article_id} >
```

Comment:
Let's use a semantic tag here too. `section` for example. `div`s are bad for accessibility.